### PR TITLE
Issue #3199 - Changed blacklisted by blocked

### PR DESCRIPTION
**Pull request by apsychogirl**
Fixes - #3199 
Changed the term blacklisted by blocked.

### CODE REVIEW

File: =2.7 [hunk 1]
```diff
@@ -0,0 +1,8 @@
++ typescript@3.8.3
```
Comment (karlcow) [inline =2.7]: This file `webcompat/=2.7` should not exist and is not part of the project. Probably a mistake, when you added or commit. 
Please remove it. :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -296,15 +296,15 @@ This is a major upgrade of our server. We are now running the project under Pyth
...
+* Fixes #2650 - Modified domain received by `is_blocked_domain` [Pull #2667](https://github.com/webcompat/webcompat.com/pull/2667)
 
 
 ## 15.0.0 - 2018-10-30
 
 * Fixes #2686 - Upgrade requests [Pull #2687](https://github.com/webcompat/webcompat.com/pull/2687)
 * Fixes #2682 - Removes unused markdown-it in issue-list.js [Pull #2683](https://github.com/webcompat/webcompat.com/pull/2683)
 * Fixes #2571 - Adds a dashboard route with the list of current dashboards [Pull #2675](https://github.com/webcompat/webcompat.com/pull/2675)
-* Fixes #2650 - Modified domain received by `is_blacklisted_domain` [Pull #2667](https://github.com/webcompat/webcompat.com/pull/2667)
+* Fixes #2650 - Modified domain received by `is_blocked_domain` [Pull #2667](https://github.com/webcompat/webcompat.com/pull/2667)
```
Comment (karlcow) [inline CHANGELOG.md]: Same here. revert to the previous version. Thanks.
```suggestion
* Fixes #2650 - Modified domain received by `is_blacklisted_domain` [Pull #2667](https://github.com/webcompat/webcompat.com/pull/2667)
```

File: package.json [hunk 3]
```diff
@@ -22,6 +22,7 @@
     "cssrecipes-utils": "^0.6.2",
     "suitcss-utils-align": "^1.0.0",
     "suitcss-utils-display": "^1.0.2",
+    "typescript": "^3.8.3",
```
Comment (karlcow) [inline package.json]: This should not be here. This is not part of the project. Please remove it. 
```suggestion
```

File: config/environment.py [hunk 4]
```diff
@@ -101,7 +101,7 @@
     See https://github.com/webcompat/css-fixme/
     for more details."""
 
-IS_BLACKLISTED_DOMAIN = ('Anonymous reporting for domain {0} '
+IS_BLOCKED_DOMAIN = ('Anonymous reporting for domain {0} '
                          'is temporarily disabled. Please contact '
                          'miket@mozilla.com '
                          'for more details.')
```
Comment (karlcow) [inline config/environment.py]: I missed this too. it's why the tests are failing.
```suggestion
IS_BLOCKED_DOMAIN = ('Anonymous reporting for domain {0} '
                     'is temporarily disabled. Please contact '
                     'miket@mozilla.com '
                     'for more details.')
```

**Comment by miketaylr** [comment]
@apsychogirl it looks like you forgot to remove the typescript package from package.json? Can you do that, then we're good to merge! :)

**Comment by karlcow** [comment]
ok i rebased in one commit. 
let's wait for the tests to run, then let's merge.

**Pull request merged**